framework/components/ecl-timelines/_timelines.scss

```diff
@@ -1,89 +1,59 @@
-/*
-Timeline
-
-Markup: html/timeline.html
+/**
+ * Timeline
+ * @define timeline
+ */
+
+$ecl-timeline-bullet-size: 14px !default;
+$ecl-timeline-border-width: 4px !default;
+
+.ecl-timeline {
+  border-left: $ecl-timeline-border-width solid map-get($ecl-colors, 'yellow-100');
+  margin: 0;
+  padding: 0;
+}
 
-Style guide timeline
-*/
+.ecl-timeline__list {
+  list-style: none;
+  padding: map-get($ecl-spacing, 'm') 0 0;
+}
 
-.timeline {
+.ecl-timeline__item {
+  margin-bottom: map-get($ecl-spacing, 'l');
+  padding-left: map-get($ecl-spacing, 'l');
   position: relative;
-  float: left;
-  margin-bottom: 1.667em;
-  padding: 2em 0 0 1.667em;
-  border-left: 4px solid map-get($ecl-colors, 'yellow-100');
-
-  .listing__item {
-    padding: 0;
-  }
-
-  .timeline__item-title {
-    margin-bottom: 0.5em;
-    line-height: 1em;
-    font-weight: bold;
-
-    &::before {
-      position: absolute;
-      left: -0.6em;
-      width: 14px;
-      height: 14px;
-      margin-top: 0.2em;
-      content: '';
-      background-color: #fff;
-      border: 3px solid map-get($ecl-colors, 'grey-100');
-      border-radius: 50%;
-    }
-  }
-
-  .timeline__text {
-    margin-top: 0.2em;
-    margin-bottom: 1em;
-
-    ul,
-    ol {
-      list-style: none !important;
-      padding-left: 0;
-    }
-
-    ul > li {
-      color: map-get($ecl-colors, 'grey-100') !important;
-    }
 
-    & > * {
-      margin-bottom: 0 !important;
-    }
+  &::before {
```

Comment:
Bullets are a little too high; there are not aligned with the title (due to its line-height)